mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result

```diff
@@ -19,3 +20,4 @@ start slave;
 include/stop_slave.inc
 include/start_slave.inc
 include/stop_slave.inc
+include/stop_dump_threads.inc
```

Comment:
From where does this come from?

Reply:
This test is not tested by Jenkins therefore it was failing before.